src/Asm89/Twig/Lint/Command/LintCommand.php

```diff
@@ -91,6 +105,10 @@ protected function execute(InputInterface $input, CliOutputInterface $output)
 
         $errors = 0;
         foreach ($files as $file) {
+            if (true === $this->isFileInExcludeList($file, $excludeList)) {
```

Comment:
The Symfony finder component supports exclusions natively. Let's use that instead:
http://api.symfony.com/2.7/Symfony/Component/Finder/Finder.html#method_exclude